README.md

```diff
@@ -20,7 +20,7 @@ To see scoverage in action check out the [samples](https://github.com/scoverage/
 
 ### Release History
 
-##### (not released yet) - 1.4.0
+#####  12th June 2019 - 1.4.0
```

Comment:
@jozic I think it was `13th June 2019`

Reply:
:) the fact that it was released matters, i guess